src/renderer/shared/ui/Identicon/Identicon.tsx

```diff
@@ -40,10 +42,10 @@ export const Identicon = ({
 
   const icon = address ? (
     <PolkadotIdenticon
-      theme={theme}
+      theme={valueTheme}
       value={address}
       size={background ? size * 0.75 : size}
-      className="pointer-events-none"
+      className="pointer-events-none rounded-full overflow-hidden"
```

Comment:
Fixed some issue?

Reply:
default it square for ethereum, we need it circle as in polkadot vault and polkadot js apps